src/test/java/de/jetwick/snacktory/ArticleTextExtractorTest.java

```diff
@@ -320,7 +320,7 @@ public void testBlogger() throws Exception {
         JResult res = extractor.extractContent(c.streamToString(getClass().getResourceAsStream("blogger.html")));
         assertTrue(res.getText(), res.getText().startsWith("Gavin King of Red Hat/Hibernate/Seam fame recently"));
         assertEquals("http://3.bp.blogspot.com/-cyMzveP3IvQ/TaR7f3qkYmI/AAAAAAAAAIk/mrChE-G0b5c/s72-c/Java.png", res.getImageUrl());
-        assertEquals("The Brain Dump: Gavin King unveils Red Hat's Java killer successor: The Ceylon Project", res.getTitle());
+        assertEquals("The Brain Dump", res.getTitle());
```

Comment:
This is an important regression. Blogger is an important blogging platform, we cannot afford to break title extraction of all blogs hosted by this service.

Before investigating extraction logic change I recommend updating the HTML dump of the page http://blog.talawah.net/2011/04/gavin-king-unviels-red-hats-top-secret.html . The HTML structure has probably evolved quite a lot since 2011 so maybe it would work fine with this new logic.

Reply:
:thinking: I tried with the version online and nothing changed, except the serving url for res.getImageUrl which is irrelevant. I need to find a solution for that thing too.